bindings/src/trade_shield/states/perpetual_order_v2.rs

```diff
@@ -0,0 +1,7 @@
+use crate::trade_shield::types::PerpetualOrderV2;
+use cw_storage_plus::Map;
+
+pub const PERPETUAL_ORDER_V2: Map<u64, PerpetualOrderV2> = Map::new("perpetual order2_v2");
```

Comment:
why do we use v2 now ? does this mean existing v1 orders will not be processed ?